reporting/spacewalk-report

```diff
@@ -158,27 +158,27 @@ if __name__ == '__main__':
             need_exit = None
             if options.info:
                 if report.synopsis is not None:
-                    print report.synopsis
+                    print(report.synopsis)
                 else:
-                    print "No synopsis for report %s." % report_name
+                    print("No synopsis for report %s." % report_name)
                 if report.description is not None:
-                    print
-                    print report.description
+                    print()
+                    print(report.description)
```

Comment:
What's wrong with:
```python
print(os.linesep, report.description)
```

Reply:
I'm not doing general code refactoring on these PRs about migration to Python 3, only the necessary fixes. I'm not taking care of fixing ugly code here.